backend/routes/boards.js

```diff
@@ -91,10 +135,14 @@ router.get("/:boardId/posts", async (req, res) => {
     );
   }
   const cards = await prisma.card.findMany({
+    orderBy: {
+      createdAt: "desc",
+    },
     where: {
-      boardId: boardId, // Replace 99 with the actual ID you want to retrieve
+      boardId: boardId,
```

Comment:
I've mentioned this comment in a previous PR review. Nice to see iterative improvements!